sale_order_type/__manifest__.py

```diff
@@ -15,6 +17,8 @@
     "Tecnativa,"
     "Agile Business Group,"
     "Niboo,"
+    "Therp BV,"
```

Comment:
@ntsirintanis Maybe too few changes to take authorship ?

Reply:
:sweat_smile: Too much opened PR's on this repo. Harsh work to decrease their number.